frontend/components/site/SiteUsers.jsx

```diff
@@ -44,6 +52,9 @@ const SiteUsers = ({ site, user }) => {
                 {' '}
                 {u.username.toLowerCase() === user.username.toLowerCase() ? '(you)' : ''}
               </td>
+              <td>
+                <ButtonLink clickHandler={handleClick(user)}>Remove user</ButtonLink>
```

Comment:
BUG: this needs to be `handleClick(u)`. `u` is the user object in the `.map` callback; `user` is the current Federalist use who is signed in. Maybe we should rename `u` to `rowUser` or something more descriptive, which might help prevent similar issues in the future.

Reply:
Also, I wonder if we should hide the button and disallow the removal on the backend for the owner of the repository. It's a little weird to delete yourself from your own site.

Reply:
Ah you're right, I had it as `u`, then moved some stuff around, then forgot to change it back...probably means something needs to be tested

Reply:
Oh, maybe you could check the arguments to the `spy` to make sure it is the expected user being passed to the handler

Reply:
You are probably right, so the only time the error in the comment above gets displayed is when the user:

* is the sole user registered in federalist
* is not the original owner of the repo

Reply:
Yeah. I think we need to keep that error for organization-owned repositories (like `18f/federalist-docs`). But for personal account-owned repos (like `jseppi/test-federalist-site`) we probably ever want the site to get to a state where the original account does not have access (unless they've totally deleted it from Federalist via the Advanced Settings).